### PR TITLE
import page tweaks

### PR DESCRIPTION
**Pull request by xithan**

### CODE REVIEW

File: mod/csv_import/set/abstract/import.rb [hunk 1]
```diff
@@ -42,7 +42,7 @@ def help_text
   end
 
   def new_view_hidden
-    hidden_tags success: { id: "_self", soft_redirect: false, view: :import }
+    hidden_tags success: { id: "_self", soft_redirect: false, redirect: true, view: :import }
```
Comment (ethn) [inline mod/csv_import/set/abstract/import.rb]: I'm thinking if our API were more like `redirect: :hard`, we'd be less likely to make this mistake.

File: mod/csv_import/set/abstract/import/02_table_row_with_company_mapping.rb [hunk 2]
```diff
@@ -13,9 +13,10 @@ def initialize csv_row, format
     @company = @match.suggestion
   end
 
-  # def render
-  #   super.merge class: "table-#{row_context}"
-  # end
+  def render
+    rendered = super
+    rendered.tap { format.add_class rendered, "_#{match_type}_match" }
```
Comment (ethn) [inline mod/csv_import/set/abstract/import/02_table_row_with_company_mapping.rb]: are we ok with using underscores in CSS classes?

File: mod/wikirate/lib/javascript/script_import_page.js.coffee [hunk 3]
```diff
@@ -8,12 +8,12 @@ decko.slotReady (slot) ->
       slot.find('.import_table input:checkbox').removeAttr 'checked'
 
   slot.find('._group_check').change (eventObject) ->
-    attr = $(this).data("group")
+    match_type = $(this).data("group")
     if $(this).is(':checked')
-      slot.find('.import_table').find('tr.table-' + attr).each ->
+      slot.find('.import_table').find("tr._#{match_type}_match").each ->
```
Comment (ethn) [inline mod/wikirate/lib/javascript/script_import_page.js.coffee]: could be much simpler.  Something like

```
match_type = $(this).data "group"
checked = $(this).is ":checked"
$('#_check_all').prop 'checked', false unless checked
slot.find('.import_table').find("tr._#{match_type}_match").each ->
  $(this).find('input:checkbox').prop 'checked', checked
```

File: mod/csv_import/set/abstract/import/02_table_row_with_company_mapping.rb (hunk 2 above)
Comment (xithan) [reply mod/csv_import/set/abstract/import/02_table_row_with_company_mapping.rb]: The trailing one is to mark it as a class for js and not for styling. We are not ok with the second one.

Comment (ethn) [reply mod/csv_import/set/abstract/import/02_table_row_with_company_mapping.rb]: 👍

File: mod/wikirate/lib/javascript/script_import_page.js.coffee (hunk 3 above)
Comment (ethn) [reply mod/wikirate/lib/javascript/script_import_page.js.coffee]: just realized "import_table" has the underscore problem.

last two lines could even be:
```
slot.find('.import_table tr._#{match_type}_match input:checkbox').prop 'checked', check
```

File: mod/wikirate/lib/javascript/script_import_page.js.coffee [hunk 4]
```diff
@@ -1,19 +1,22 @@
...
-      $('._group_check').prop 'checked', true
-      slot.find('.import_table input:checkbox[disabled!="disabled"]').prop 'checked', true
-    else
-      $('._group_check').removeAttr 'checked'
-      slot.find('.import_table input:checkbox').removeAttr 'checked'
+  selectImportRows = (checked, match_type) ->
+    selector = "._import-table"
+    selector += " tr._#{match_type}-match" if match_type?
+    selector += " input:checkbox[disabled!=\"disabled\"]"
+    slot.find(selector).prop 'checked', checked
```
Comment (ethn) [inline mod/wikirate/lib/javascript/script_import_page.js.coffee]: I like!

**Pull request merged**